2023-10-18/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt"
+
+func concatenate(a, b string) string {
+	return a + b
+}
+
+func main() {
+	a := "el"
+	b := "Jo"
+
+	a, b = b, a
```

Comment:
Assigning variable a to variable b and variable b to variable a 

meaning, today we actually flip the names of the variables. meaning after this assignment we now have
`a := "Jo"
b := "el"`